src/exo/LoopIR_scheduling.py

```diff
@@ -320,6 +321,12 @@ def DoReorderStmt(f_cursor, s_cursor):
     return ir, fwd
 
 
+def DoParallelizeLoop(loop_cursor):
+    # TODO: Check commutativiity of loop body
+    Check_ParallelizeLoop(loop_cursor.get_root(), loop_cursor._node)
+    return loop_cursor._child_node("config")._replace(LoopIR.Par())
```

Comment:
I stared at this for 30 seconds wondering why configuration is replaced here, can we use other word for "loop_config"? How about "is_par" or "loop_mode"?

Reply:
config will be sooo confusing in Exo

Reply:
Can't we rely on inheritance here?
For = Seq()
          | Par()
          attributes(sym iter, expr lo, expr hi, stmt* body, loop_config config)

This is also to simplify code. I would imagine being able to immediately write:
`isinstance(node, Par)` when I want to check if some random node is a parallel loop as opposed to writing `isinstance(node, For)` and `isinstance(node.config, Par)`

Reply:
I don't believe ASDL allows for that kind of inheritance, based on what I could tell from its grammar.